test/index.js

```diff
@@ -0,0 +1,1072 @@
+var path = require('path')
+var test = require('tape')
+var rimraf = require('rimraf')
+
+var Clocker = require('../lib/index')
+
+test('start', function (t) {
+  t.plan(4)
+
+  t.test('without arguments', function (t) {
+    var clocker = initialize()
+
+    clocker.start(function (err, key) {
+      t.notOk(err)
+      t.ok(key, 'key is generated')
+
+      clocker.close(function () {
+        t.end()
+      })
+    })
+  })
+
+  t.test('date given', function (t) {
+    t.plan(2)
+
+    t.test('as Date', function (t) {
+      var clocker = initialize()
+
+      var date = new Date()
```

Comment:
This makes the test non-reproducible. I would avoid this by setting the date to a specific day.

Reply:
I added the actual date.